mypy/test/testcmdline.py

```diff
@@ -58,6 +58,14 @@ def test_python_cmdline(testcase: DataDrivenTestCase) -> None:
     outb = process.stdout.read()
     # Split output into lines.
     out = [s.rstrip('\n\r') for s in str(outb, 'utf8').splitlines()]
+
+    is_running_in_py_charm = "PYCHARM_HOSTED" in os.environ
+    if is_running_in_py_charm:
```

Comment:
I don't think you need to create a variable here. I'd just do `if "PYCHARM_HOSTED" in os.environ: ...`